corehq/apps/style/templates/style/includes/analytics_google.html

```diff
@@ -91,17 +91,26 @@
      * Avoids duplicating checks that GA has been loaded.
      */
     function ga_track_event() {
-        var args = ['send', 'event'];
-        for(var i = 0; i < arguments.length; i++) {
-            args.push(arguments[i]);
+        if (window.ga && ga.create){
+            // Check that Google Analytics is not blocked by apps such as Ghostery
+            var args = ['send', 'event'];
+            for(var i = 0; i < arguments.length; i++) {
+                args.push(arguments[i]);
+            }
+            // check the last from the right
+            // rather than the 3rd or 4th from the left, because it's variable
+            // https://developers.google.com/analytics/devguides/collection/analyticsjs/events#overview
+            var lastArg = arguments[arguments.length - 1];
+            if (lastArg && lastArg.hitCallback && window.event) {
+                window.event.preventDefault();
+            }
+            ga.apply(null, args);
         }
-        // check the last from the right
-        // rather than the 3rd or 4th from the left, because it's variable
-        // https://developers.google.com/analytics/devguides/collection/analyticsjs/events#overview
-        var lastArg = arguments[arguments.length - 1];
-        if (lastArg && lastArg.hitCallback && window.event) {
-            window.event.preventDefault();
+        else{
+            // If GA is blocked, run the hitCallback
+            var callBack = lastArg.hitCallback;
```

Comment:
ugh! It worked and I didn't pay attention, will fix that (wondering how it worked)